js/coinbase.js

```diff
@@ -1032,34 +1032,24 @@ module.exports = class coinbase extends Exchange {
          */
         await this.loadMarkets ();
         symbols = this.marketSymbols (symbols);
-        const request = {
-            // 'currency': 'USD',
-        };
-        const response = await this.v2PublicGetExchangeRates (this.extend (request, params));
-        //
-        //     {
-        //         "data":{
-        //             "currency":"USD",
-        //             "rates":{
-        //                 "AED":"3.6731",
-        //                 "AFN":"103.163942",
-        //                 "ALL":"106.973038",
-        //             }
-        //         }
-        //     }
-        //
-        const data = this.safeValue (response, 'data', {});
-        const rates = this.safeValue (data, 'rates', {});
-        const quoteId = this.safeString (data, 'currency');
+        let symbol = undefined;
         const result = {};
-        const baseIds = Object.keys (rates);
-        const delimiter = '-';
-        for (let i = 0; i < baseIds.length; i++) {
-            const baseId = baseIds[i];
-            const marketId = baseId + delimiter + quoteId;
-            const market = this.safeMarket (marketId, undefined, delimiter);
-            const symbol = market['symbol'];
-            result[symbol] = this.parseTicker (rates[baseId], market);
+        if (symbols !== undefined) {
+            for (let i = 0; i < symbols.length; i++) {
+                symbol = symbols[i];
+                result[symbol] = await this.fetchTicker (symbol);
+            }
+        } else {
+            const fetchTickerMethod = this.safeString (this.options, 'fetchTicker');
+            if (fetchTickerMethod !== 'fetchTickerV3') {
+                this.checkRequiredArgument ('fetchTickers', symbols, 'symbols');
+            }
+            const markets = await this.fetchMarkets ();
```

Comment:
@Dan-krm unfortunately I don't think we can do this, we should refrain from making more than 1 req per unified method as much as we can. In this case i would suggest leaving fetchTickers with v2, the user can easily do this on his end (call `fetchTicker` multiple times)

Reply:
Sounds good, I restored the v2 fetchTickers and added a new version of fetchTickersV3